packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -28,10 +30,24 @@ const ConnectorRow = ({ type, connector, onClickSetup }: Props) => {
     return t('connectors.type.social');
   }, [type, t]);
 
+  const iconPlaceholder = useMemo(() => {
+    if (type === ConnectorType.Email) {
+      return emailConnectorIcon;
+    }
+
+    if (type === ConnectorType.SMS) {
+      return smsConnectorIcon;
```

Comment:
Or I think this icon placeholder logic should be put into the `ConnectorName` component. Since `ConnectorName` has the connector object as a param, it should be able to get connector type and calculate each placeholder icon